### DISCUSSION TITLE
Roadmap for additional platform support

### DISCUSSION THREAD

**Discussion by 2fsbb9op** [webrcade/webrcade#31] [Ideas]
> The quantity and type of applications will continue to expand over time.

What other emulators are planned?  My top three wish list:

1. [ColecoVision](https://en.wikipedia.org/wiki/ColecoVision)
2. [Intellivision](https://en.wikipedia.org/wiki/Intellivision)
3. [Atari 5200](https://en.wikipedia.org/wiki/Atari_5200)

Thanks for the great retro gaming experience!

**Comment by PaulLuigi** [comment]
My tops are N64 and PS1.  When I play Webrcade, I do so with a nice controller that has dual stick and is perfect for more advanced platforms, the kind that would probably still run great on new iPhones/iPads.

**Comment by raz0red** [comment]
N64 is the next release (post 0.0.4). So, shouldn't be too much longer.
